requirements.txt

```diff
@@ -10,7 +10,7 @@ pyyaml==5.4.1
 bidict==0.21.2
 click==7.1.2
 dnspython==1.16.0
-greenlet==1.0.0
+greenlet==1.1.2
```

Comment:
Is there a reason to use 1.1.2 instead of the current 1.1.3?

Reply:
The last documented release is 1.1.2:

https://greenlet.readthedocs.io/en/latest/changes.html#id2

I don't want to upgrade to 1.1.3 without information about what's included.

Reply:
The upgrade to 1.1.3 supports Python 3.11: https://greenlet.readthedocs.io/en/stable/changes.html#id1 (from the greenlet GitHub issue).